src/strawman/collections/CollectionStrawMan6.scala

```diff
@@ -0,0 +1,980 @@
+package strawman.collections
+
+import Predef.{augmentString => _, wrapString => _, _}
+import scala.reflect.ClassTag
+import annotation.unchecked.uncheckedVariance
+import annotation.tailrec
+
+class LowPriority {
+  import CollectionStrawMan6._
+
+  /** Convert array to iterable via view. Lower priority than ArrayOps */
+  implicit def arrayToView[T](xs: Array[T]): ArrayView[T] = new ArrayView[T](xs)
```

Comment:
Then they would not be considered in this scenario:

    val x: Seq[Int] = Array(1, 2, 3)